dashboards/ruby_vm/main.json

```
@@ -142,6 +142,31 @@
           }
         ],
         "type": "timeseries"
+      },
+      {
+        "title": "GC time",
+        "description": "The Garbage Collection time measured. (Not enabled by default.)",
```

```suggestion
        "description": "The Garbage Collection time measured, if Ruby GC profiling is enabled",
```

Maybe something like this will do, given there's no support in the description for links or code blocks? Googling "enable Ruby GC profiling" tells you how to do it.